setup.py

```diff
@@ -21,8 +21,10 @@
 # https://github.com/kivy/buildozer/issues/722
 install_reqs = [
     'appdirs', 'colorama>=0.3.3', 'jinja2', 'six',
-    'enum34; python_version<"3.4"', 'sh>=1.10; sys_platform!="nt"'
+    'enum34; python_version<"3.4"', 'sh>=1.10; sys_platform!="nt"',
+    'pep517', 'pytoml', 'virtualenv'
```

Comment:
:sob:

Reply:
:grimacing:

Reply:
I am wondering, virtualenv was previously already used in other parts but this was never an install req. It seems to me like that was actually an oversight and it should be there, or am I missing something? Should this go away?

I mean I'm not actually ever directly importing it, but I am calling it via python -m virtualenv (and most of the other code via the virtualenv executable, which I believe actually should be refactored into the same call style because it can break with user local installs, but that's another issue)